static/js/upload_widget.js

```diff
@@ -125,9 +125,10 @@ exports.build_direct_upload_widget = function (
     max_file_upload_size = max_file_upload_size || default_max_file_size;
     function accept() {
         input_error.hide();
-        if (upload_button.hasClass("night-settings")) {
+        const realm_logo_section = upload_button.closest(".realm-logo-section");
+        if ($(realm_logo_section).attr('id') === "night-logo-section") {
```

Comment:
I think we do not require `$` here, `realm_logo_section.attr('id')` will work.

Reply:
ok @sahil839